src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -23,6 +23,8 @@ internal static class Utils
         // Keeping this number low will preserve some stack space
         internal const int XmlDsigSearchDepth = 20;
 
+        internal static readonly Encoding DefaultEncoding = new UTF8Encoding(false);
```

Comment:
Is Utils.DefaultEncoding ever used for anything that might result in a preamble being emitted?  If not, why not just use Encoding.UTF8 at all the call sites?